source/install/config-proxy-nginx.rst

```diff
@@ -71,20 +71,28 @@ NGINX is configured using a file in the ``/etc/nginx/sites-available`` directory
 
 6. Restart NGINX.
 
-  On Ubuntu 14.04 and RHEL 6.6:
-  
-  ``sudo service nginx restart``
-  
-  On Ubuntu 16.04 and RHEL 7.1:
-  
-  ``sudo systemctl restart nginx``
+  On Ubuntu 14.04 and RHEL 6.6: ``sudo service nginx restart``
+
+  On Ubuntu 16.04 and RHEL 7.1: ``sudo systemctl restart nginx``
 
 7. Verify that you can see Mattermost through the proxy.
 
   ``curl http://localhost``
-  
+
   If everything is working, you will see the HTML for the Mattermost signup page.
 
-**What to do next**
+8. Make sure that Mattermost accepts connections only from the proxy.
```

Comment:
Can we make this step optional.  It is more secure like this, but there are times when you may want to connect to the app server directly to administer the system console of each server when in HA mode for example.